Sources/Plasma/PubUtilLib/plVault/plVaultClientApi.cpp

```diff
@@ -414,16 +414,17 @@ static void FetchRefOwners (
     NetVaultNodeRef *           refs,
     unsigned                    refCount
 ) {
-    TArray<unsigned> ownerIds;
-    {   for (unsigned i = 0; i < refCount; ++i)
-            if (unsigned ownerId = refs[i].ownerId)
-                ownerIds.Add(ownerId);
+    std::vector<unsigned> ownerIds;
+    for (unsigned i = 0; i < refCount; ++i) {
+        if (unsigned ownerId = refs[i].ownerId)
+            ownerIds.emplace_back(ownerId);
     }
-    std::sort(ownerIds.Ptr(), ownerIds.Term());
+    std::sort(ownerIds.begin(), ownerIds.end());
     NetVaultNode templateNode;
     templateNode.SetNodeType(plVault::kNodeType_PlayerInfo);
-    {   unsigned prevId = 0;
-        for (unsigned i = 0; i < ownerIds.Count(); ++i) {
+    {
+        unsigned prevId = 0;
+        for (unsigned i = 0; i < ownerIds.size(); ++i) {
```

Comment:
Might as well go ahead with this being a range loop.

Reply:
In this loop, `i` is used to index both `ownerIds` and `refs`

Reply:
`size_t` in that case?